pkg/eestream/decode.go

```diff
@@ -101,18 +101,24 @@ func (dr *decodedReader) Close() error {
 	// avoid double close of readers
 	errorThreshold := len(dr.readers) - dr.scheme.RequiredCount()
 	dr.close.Do(func() {
-		var errlist errs.Group
-		// close the readers
+		var errlist errgroup.Group
```

Comment:
`errgroup` returns only the first error instead of all of them

Reply:
do we have something that will fit here or I just need to use for example `sync.WaitGroup`?

Reply:
As far as I know, no, but might makes sense to implement one.